packages/utils/telemetry-utils/src/debugLogger.ts

```diff
@@ -54,12 +54,19 @@ export class DebugLogger extends TelemetryLogger {
         }
 
         const multiSinkLogger = new MultiSinkLogger(undefined, properties);
-        multiSinkLogger.addLogger(DebugLogger.create(namespace));
+        multiSinkLogger.addLogger(DebugLogger.create(namespace, this.tryGetBaseLoggerProps(baseLogger)));
         multiSinkLogger.addLogger(ChildLogger.create(baseLogger, namespace));
 
         return multiSinkLogger;
     }
 
+    private static tryGetBaseLoggerProps(baseLogger?: ITelemetryBaseLogger) {
+        if(baseLogger instanceof TelemetryLogger) {
+            return (baseLogger as any as {properties: ITelemetryLoggerPropertyBags}).properties;
```

Comment:
this solves a problem more related to our tests, the debug logger didn't get props off the incoming logger, so didn't have things like runid in the stress case